e2e/fixtures/api.ts

```diff
@@ -0,0 +1,26 @@
+import { APIRequestContext, PlaywrightWorkerArgs, WorkerFixture } from '@playwright/test';
+
+/**
+ * A fixture which initializes an [`APIRequestContext`](https://playwright.dev/docs/api/class-apirequestcontext)
+ * that is bound to the configured OpenMRS API server. The context is automatically authenticated
+ * using the configured admin account.
+ *
+ * Use the request context like this:
+ * ```ts
+ * test('your test', async ({ api }) => {
+ *   const res = await api.get('patient/1234');
+ *   await expect(res.ok()).toBeTruthy();
+ * });
+ * ```
+ */
+export const api: WorkerFixture<APIRequestContext, PlaywrightWorkerArgs> = async ({ playwright }, use) => {
+  const ctx = await playwright.request.newContext({
+    baseURL: `${process.env.E2E_BASE_URL}/ws/rest/v1/`,
```

Comment:
In patient chart, we also use the `/ws/fhir2/R4/` (FHIR endpoints) too.
For e.g. the hook for fetching the patient details `usePatient` actually fetches from the FHIR endpoint.
Thanks!

Reply:
Are the FHIR endpoints used for any other operations besides fetching data? For example, do we use them for POST, PUT, or DELETE operations?

Reply:
> In patient chart, we also use the `/ws/fhir2/R4/` (FHIR endpoints) too. For e.g. the hook for fetching the patient details `usePatient` actually fetches from the FHIR endpoint. Thanks!

As for now, we're using this as an initial PR and if we need one in the future I'll include it.

Reply:
Using FHIR sounds like a good idea since we're mainly using FHIR endpoints. It'll help us reuse the same data structures. 
> As for now, we're using this as an initial PR and if we need one in the future I'll include it.

Yeah, @RandilaP, we can integrate it as we move forward.